ecosystem/sep-0006.md

```diff
@@ -1121,6 +1121,8 @@ Name | Type | Description
 `refunds` | object | (optional) An object describing any on or off-chain refund associated with this transaction. The schema for this object is defined in the [Refunds Object Schema](#refunds-object-schema) section below.
 `required_info_message` | string | (optional) A human-readable message indicating any errors that require updated information from the user.
 `required_info_updates` | object | (optional) A set of fields that require update from the user described in the same format as [/info](#info).  This field is only relevant when `status` is `pending_transaction_info_update`.
+`customer_info_type` | string | (optional) The [SEP-12 type](sep-0012.md#type-specification) of customer information that is required to complete the transaction. This field is only relevant when `status` is `pending_customer_info_update`. Wallets should use this field to determine which fields to request from the user.
+`how` | string | (optional) A human-readable message indicating how the user should complete the transaction. This field is only required if `how` was not included in the response of a `deposit` request.
```

Comment:
Rather than port `how` to the `/transaction(s)` response, I'd rather move away from having the anchor produce human-readable content that the anchor has no control over.

Instead, we should create a object that contain a set of SEP-9 account-related fields, providing information about the business' off-chain account in the case of deposit and on-chain account in the case of withdraw.

Reply:
The plan is to replace `how` with something more structured but I was thinking of addressing that in another PR. Is that okay with you?

Reply:
I think it makes sense to do it in one PR, otherwise we will have ab in-between state we are not satisfied with